packages/probing-frontend/src/config/constants.ts

```diff
@@ -30,3 +31,14 @@ export const RootLink = {
   ariaLabel: 'Vai al sito di PagoPA S.p.A.',
   title: 'Vai al sito di PagoPA S.p.A.',
 }
+
+export const headLabels = (t: TFunction<'common', 'table'>): Array<string> => {
+  return [
+    t('eServiceName'),
+    t('version'),
+    t('producer'),
+    t('eServiceStatus'),
+    t('lastDetectionDate'),
+    '',
+  ]
+}
```

Comment:
is this constant used in more than one file? If yes, keep it here but give this constant a more descriptive name (ex: `monitoringTableHeadLabels`), if not put it inside the component/file